assets/src/blocks/editorIndex.js

```diff
@@ -1,10 +1,14 @@
 import {registerPostsListBlock} from './PostsList';
 import {registerActionsList} from './ActionsList';
+import {registerSubmenuBlock} from './Submenu/SubmenuBlock';
 
 wp.domReady(() => {
   // Make sure to unregister the posts-list native variation before registering planet4-blocks/posts-list-block
   wp.blocks.unregisterBlockVariation('core/query', 'posts-list');
 
+  // Blocks
```

Comment:
I think that so far we only use this file to load the beta blocks, so maybe this means that the Submenu block won't be available when the beta blocks are disabled? Otherwise, does it mean the beta blocks are always available? We should double-check

Reply:
Right, that makes sense! Thanks for the explanation 🙂